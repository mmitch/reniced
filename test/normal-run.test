#!/bin/sh

# mock ps output
PATH=".:${PATH+:}${PATH}"
export PATH

# run test in no-op mode
../reniced -n test.conf
