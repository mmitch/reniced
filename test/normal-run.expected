would set priority of 201 to 5
would set priority of 2445 to 5
would set IO priority for pid 19966 to class 3
would set priority of 1234567 to 3
would adjust OOM setting of pid 1234567 to -10
